examples/durable/DurableApp/FanOutFanInOrchestrator/run.ps1

```diff
@@ -9,7 +9,7 @@ $parallelTasks =
         Invoke-ActivityFunction -FunctionName 'SayHello' -Input $Name -NoWait
     }
 
-$output = Wait-DurableTask -Task $parallelTasks
+$output = Wait-ActivityFunction -Task $parallelTasks
```

Comment:
I think we should actually keep `Wait-DurableTask` in our examples and tests. When I asked for an alias, I was thinking about _external users_ code. Public Preview has been announced today, so external users will start writing code using `Wait-ActivityFunction` and we don't want to break it suddenly without a strong reason and an appropriate announcement. However, we do want to rename the command, so the code we directly control (examples and tests) should reflect this intention.